tutorials/getting_started/first_steps.md

```diff
@@ -31,13 +31,18 @@ Start by setting your language, region, and time zone. You can also set your loc
 ### Install add-ons
 
 You now have the option to install openHAB add-ons.
-Do this if you already know that you'll need specific add-ons (e.g. if you're upgrading from an older openHAB system), or skip this step if you prefer to install add-ons individually.
+The openHAB system does an automatic scan to discover home automation equipment and software on your PC and LAN, and based the result it recommends suggested add-ons to install. 
+You can modify the list of add-ons.
+However, do this only if you already know that you'll need specific add-ons (e.g. if you're upgrading from an older openHAB system).
+Or skip this step if you prefer to install add-ons individually later.
 You can always install/remove add-ons in the future, enabling you to modify and improve your system over time.
 
-![addons wizard](images/wizard_addons.png)
+![addons wizard](https://github.com/openhab/openhab-docs/assets/893994/ecbd893b-3b95-4fd6-a89c-6f9a87313229)
```

Comment:
Is that a stable link? We are we using this particular link?

Reply:
Same question as for my other PR => @Confectrician

Reply:
Here is the image

![image](https://github.com/openhab/openhab-docs/assets/893994/7557d719-c4d8-43fd-820e-9b5a886d4c44)

Reply:
I fixed the image link